components/renku_data_services/project/api.spec.yaml

```
@@ -335,6 +340,8 @@ components:
           default: "private"
         description:
           $ref: "#/components/schemas/Description"
+        keywords:
+          $ref: "#/components/schemas/KeywordsList"
```

```suggestion
          $ref: "#/components/schemas/KeywordsList"
          default: []
```

If you add this the dataclass generated from the apispec will have a deafult value of [] rather than None. But up to you if you prefer this or not. I find it easier if a list is initiated empty as default rather than None. It is easier to keep mypy happy with a single type rather than with `None | list[str]`. But it is a matter of taste.